src/components/molecules/Visualizer/Plugin/hooks.ts

```diff
@@ -104,9 +104,9 @@ export default function ({
 
   useEffect(() => {
     const visible = shownPluginPopupInfo?.id === (widget?.id ?? block?.id);
-    if (popupVisible.current !== visible) {
-      popupVisible.current = visible;
-      if (!popupVisible.current) {
+    if (popupVisible !== visible) {
+      setPopupVisibility(visible);
+      if (!visible) {
```

Comment:
Shouldn't this be popupVisible, not visible?

Reply:
Please check the previous reply.